redis/src/cluster_async/mod.rs

```diff
@@ -94,17 +94,21 @@ where
 type ConnectionFuture<C> = future::Shared<BoxFuture<'static, C>>;
 type ConnectionMap<C> = HashMap<String, ConnectionFuture<C>>;
 
-struct ClusterConnInner<C> {
-    connections: ConnectionMap<C>,
+struct ConnectionsAndMetadata<C> {
```

Comment:
Minor, but a more terse name here would be great. `ConnsMeta` is perhaps not appropriate but saves us 13 columns.

Reply:
"Core" is 4 characters. I'll take it!

Reply:
renamed.